### DISCUSSION TITLE
Gluetun+SABNZBD GUI status utilitys wont work when behind a VPN

### DISCUSSION THREAD

**Discussion by INeedHelp321** [qdm12/gluetun#2239] [Q&A]
As title says when Sabnzbd is behind Gluetun everything else works except interface status window where I should see IP adresses and also can test folder speeds. This is probably caused by something I just dont realize about docker-container networking, Localhost not connecting properly or something like nameserver not connecting etc... Attached debug log from sabnzbd and log from gluetun hopefully helps to point me to right direction. - Thank you! @qdm12 

![image](https://github.com/qdm12/gluetun/assets/82748918/30503213-b314-4fa3-919d-3182d298a034)

[message.txt](https://github.com/qdm12/gluetun/files/15122081/message.txt)

[gluetun log.txt](https://github.com/qdm12/gluetun/files/15128623/gluetun.log.txt)

[compose config.txt](https://github.com/qdm12/gluetun/files/15129672/compose.config.txt)

edit. Tried adding - FIREWALL_OUTBOUND_SUBNETS=192.168.1.0/24 without making a difference.

**Comment by frepke** [comment]
Hi @INeedHelp321,

Since this is the Gluetun repository and not the Sabnzbd repository, it's probably more helpful to also share the Gluetun logs and Gluetun configuration instead of the Sabnzbd log.

kr.,
Frepke

**Reply by INeedHelp321** [reply]
Hi @frepke very much true.. Addded to the first post.

**Comment by frepke** [comment]
You exposed your credentials in the config file.

**Reply by INeedHelp321** [reply]
I did, little tired this morning. Changed password and username on vpn provider. Thanks for pointing this out kindly!

**Comment by frepke** [comment]
Did you try to clear your browser cache?
Can't see anything weird in your composefile :(

**Reply by INeedHelp321** [reply]
Yes I tried. Also tried with other browser. I think remembering fixing this somehow 1-2years ago with openvpn, but I did not find my documentation about how I did this. Maybe something with DNS/nameserver not working properly. Remember doing something with resolv.conf but cant recall what and why back then or was it even related to this.

Tested with older version of gluetun .33 .35 and newest .38 as someone said only .35 worked for them but no help for me.
